modules/appeals_api/spec/requests/v2/supplemental_claims_controller_spec.rb

```diff
@@ -70,20 +70,6 @@ def new_base_path(path)
         expect(parsed['data']['attributes']['status']).to eq('pending')
         expect(parsed.dig('data', 'attributes', 'formData')).to be_a Hash
       end
-
-      it 'behaves the same on new path' do
-        Timecop.freeze(Time.current) do
-          post(path, params: data, headers: headers)
-          orig_path_response = JSON.parse(response.body)
-          orig_path_response['data']['id'] = 'ignored'
-
-          post(new_base_path('forms/200995'), params: data, headers: headers)
-          new_path_response = JSON.parse(response.body)
-          new_path_response['data']['id'] = 'ignored'
-
-          expect(new_path_response).to match_array orig_path_response
```

Comment:
We're losing this check that the responses are the exact same and replacing it with the shared examples for oauth. My reasoning for removing this was that we're using the same actions to respond on both the old and new paths, so it seemed to me that checking that the request succeeded was enough.

But that requires knowledge of how the new controller inherits from the old one, so I can see the reasoning for keeping these sorts of tests, too. Let me know if it makes more sense to everyone to keep there more specific checks, and if so I'll restore this and add a few more like it.

Reply:
I've gone back on this and added a new commit that adds several new test cases to compare the bodies of the success responses for oauth/non-oauth endpoints